### ISSUE TITLE
Sdiebar Icons Not Showing Up.

### ISSUE THREAD

**Issue by DaximusPrime** [Hutchy68/pivot#43]
### Setup

**- Browsers and versions:**
Mozilla Firefox
**- MediaWiki version:**
1.27.3
**- Database version:**
**- PHP version:**
5.6.31 (cgi-fcgi)
**- Pivot version:**
1.0.5
**- Extension version (if connected to the issue):**

### Issue
I am having an issue getting the sidebar icons for Recent Changes, Random Page, as well as the icons on the top of the for Actions and the slide out side bar on the right side. None of the little icons show up at all.

**Comment by Hutchy68** [comment]
Please check, https://pivot.wikiproject.net/wiki/FAQs the first FAQ.

**Comment by DaximusPrime** [comment]
Is there any way to remove the icons completely and just not have it show the broken links?

**Comment by Hutchy68** [comment]
No, unfortunately not. 

Do you have a URL so I can see what is going on.

**Comment by Celdecea** [comment]
Just in case this helps anyone.

My problem with the icons was that I uploaded the folder as "Pivot" with a capital P while the icons were being requested from the all lowercase "pivot." Everything else installs correctly and it appears to function except the font files get served up as error pages so they look like empty squares.

Fixing the folder name broke stuff, so I just uploaded a lowercase copy of the folder and all is magic.

Nice skin, I like the simplicity.
